e2e/README.md

```diff
@@ -22,11 +22,58 @@ nodes can be identified by the following container names (kube-master, kube-node
 All tests with the name `SmokeSuite` in the method will execute and the results will be of a junit
 type output to the `e2e/results` directory.
 
+## Run Tests
+Rook Framework as several configuration options, that can be used to run install rook and run tests, or just run 
+tests against rook that is already installed.
+  
+#### Test parameters
+The following parameters are available while running tests
+
+ Parameter | Description | Default value
+ --- |--- | --- 
+rook_platform| platform rook needs to be installed on  | kubernetes
+k8s_version  | version of Kubernetes to be installed  | v1.6
+rook_version | rook version/tag to be installed | master-latest 
+install_rook | skips installing rook(if already installed) | true
```

Comment:
what about renaming this flag to `skip-install-rook`? the default would then be `false`